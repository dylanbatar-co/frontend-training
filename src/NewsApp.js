import React from 'react';
import { Alert } from './components/Alert/Alert';
import { Header } from './components/Header/Header';
import { Banner, ContactUs, News } from './components/Sections';

export const NewsApp = () => {
  return (
    <>
      <Header />
      <main className="mt-8">
        <Alert />
        <News />
        <Banner />
        <ContactUs />
      </main>
    </>
  );
};
