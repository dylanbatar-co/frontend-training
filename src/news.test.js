import { shallow } from 'enzyme';
import { NewsApp } from './NewsApp';

describe('Main app', () => {
  it('NewsApp component snapshot', () => {
    const component = shallow(<NewsApp />);
    expect(component).toMatchSnapshot();
  });
});
