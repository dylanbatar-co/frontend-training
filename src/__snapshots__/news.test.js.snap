// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main app NewsApp component snapshot 1`] = `
<Fragment>
  <Header />
  <main
    className="mt-10 mb-8"
  >
    <Alert />
    <News />
    <Banner />
    <ContactUs />
  </main>
</Fragment>
`;
